Jellyfin.Plugin.AniDB/Providers/equals_check.cs

```diff
@@ -68,13 +72,13 @@ public static string FuzzyRegexEscape(string a)
 
             // whitespace
             a = a.Replace(@"\ ", ".?.?.?");
```

Comment:
Just noticed, this is just a escaped whitespace regex replace, no?
Is this nnot also covered by the Regex Class `\s` in `_whitespaceRegex`?

Reply:
Yes, the `\s` only replaces the actual white space, not the escape character. An escaped white space would become `\.?.?.?` instead of `.?.?.?`.

Reply:
ah, i see. interesting to see / imagine file / series names with escaped spaces in them :laughing:

Reply:
`Regex.Escape` escapes spaces.
The `\s` replacement is only needed for non standard spaces, because all normal spaces get escaped `Regex.Escape` and then replaced by this.

Reply:
`a = a.Replace(@"\ ", ".?.?.?");` is just a normal Replace, not a Regex.

Reply:
> `a = a.Replace(@"\ ", ".?.?.?");` is just a normal Replace, not a Regex.

I knew I was missing the obvious :man_facepalming:, sorry for the confusion I must have read it as regex while glancing over again :sweat_smile: